client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -153,7 +153,7 @@ export class PluginAutoUpdateToggle extends Component {
 			] ),
 			getDisabledInfo = this.getDisabledInfo(),
 			label = translate( 'Autoupdates', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **45** times:
`translate( 'Autoupdates', { context: 'this goes next to an icon that displays if the plugin has "autoupdates" active'} )` *ES Score: 24*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/emilyaudela-patch-1)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).